Makefile-docker

```diff
@@ -52,37 +49,26 @@ check_django: ## check if the django app is configured properly
 .PHONY: check
 check: check_files check_olympia_user check_debian_packages check_pip_packages check_django
 
+.PHONY: data_dump
+data_dump:
+	./manage.py data_dump $(ARGS)
+
+.PHONY: data_load
+data_load:
+	./manage.py data_load $(ARGS)
+
 .PHONY: initialize_db
 initialize_db: ## create a new database
 	rm -rf ./user-media/* ./tmp/*
-	$(PYTHON_COMMAND) manage.py create_db --force
-	$(PYTHON_COMMAND) manage.py migrate --noinput
-	$(PYTHON_COMMAND) manage.py loaddata initial.json
-	$(PYTHON_COMMAND) manage.py import_prod_versions
-	# The superuser needs to have a mozilla.com address for admin tools access
-	$(PYTHON_COMMAND) manage.py createsuperuser \
-		--no-input \
-		--username "local_admin" \
-		--email "local_admin@mozilla.com"
-	$(PYTHON_COMMAND) manage.py loaddata zadmin/users
+	# Seed the database with initial data
+	./manage.py data_seed
+	# Load the data from the initial dump
+	$(MAKE) data_load ARGS="--name _init"
```

Comment:
it's unfortunate we hard-code `_init` here _and_ in amo/management/base.py

Reply:
I actually don't need that line because data_seed already loads the data. so that's totally redundant.

Reply:
Fixed